### DISCUSSION TITLE
[공지사항] ※2021-01-16 수업 후 공지사항※

### DISCUSSION THREAD

**Discussion by yoomambo** [4Moyede/HexatonClass-01#124] [00_공지사항]
여러분 안녕하세요

멘토 유혁준, 강형구입니다.

## 프로젝트 팀 구성

우선 팀원 구성 말씀드릴게요. 여러분들이 순위 적어주셨던 대로 구성하였고, 대부분 1순위로 적은 순으로 팀 구성하였습니다. 팀 변경이 있다면, 멘토한테 말씀해주시길 바랍니다. 뭐... 팀을 1개로 합쳐도 상관없습니다!

### 구성원

- 지우 + 재범 + 경훈
- 덕규 + 성준

팀구성은 이렇게 되었습니다. 팀원간의 회의를 통해 풀고싶은 문제를 명확하게 하시면 됩니다. 주제를 변경 하여도 좋고, 서로의 아이디어를 모아 더욱더 발전 시켜도 좋습니다.

### 면담 시간

팀 정해진대로 프로젝트를 좀 더 구체화 시켜볼 필요가 있을 것 같아 면담을 하고자 합니다.

1. 1/17일 일요일 20:00 - 21:00
2. 1/17일 일요일 21:00 - 22:00
3. 1/17일 일요일 22:00 - 23:00

3가지 중에 고르셔서 팀 중 한명만 댓글로 말씀해주시면 됩니다.

## 과제 (1/21 목요일 오후 6시까지 제출)

1. Notion에 지금까지 수업들었던 부분, 수업에서 배웠던 부분 페이지에 정리해서 공유하기. 양식은 없으나 진짜 공부한다고 생각하고 만들어보기. 
2. 짜여진 팀, 논의 후 구성원 중 한명이 proposal 구체적으로 적기. 이 proposal을 보더라도 사람들이 이해안되서 질문 안할 정도로 자세히 적기 ex) 데이터가 어떤 데이터인지, 정확히 누가 타겟층인지, 어떤 아웃풋을 낼 것인지 (대략적인 그림이나 구조)
3. 수업때 proposal 발표 할 거니까 발표준비! (한 명만 발표준비하면 됩니당)
4. 형구 멘토님이 내준 계산기 class 문제 풀어서 .py 파일로 만들어서 github에 폴더 이름 03에 제출
5. 다른 멘티들 코드 보고 멘티들마다 리뷰하고, 멘토 파일도 리뷰 해서 과제리뷰2주차에 댓글로 남기기

이번에는 과제가 많이 없습니다! 없는 이유는 지금까지 공부했던 부분들을 아마 정리할 시간이 없었을 겁니다. 그래서 프로젝트 준비하시면서 여러분들이 지금까지 정리하지 못했던 부분들을 챙기는 시간을 가지시길 바랍니다.

**Comment by ghost** [comment]
확인했습니다

**Comment by yoomambo** [comment]
![image](https://user-images.githubusercontent.com/54018187/104815029-7d256800-5855-11eb-9562-c3c46287ce02.png)
그리고 지금 지난주 이슈 정리랑 과제 리뷰 안하신분들 빨리하세요~~~

@JaebeomHan @lsj0822 @NA-DK @jiwoo4424 @dlrudgnsdlek

**Comment by dlrudgnsdlek** [comment]
확인했습니당

**Comment by jiwoo4424** [comment]
저희 오늘 밤 10시-11시 타임 줌들어갈게요!
